scripts/update-fake-master.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kinit -k -t $KEYTAB $PRINCIPLE
+
+# Use the directory relative to this Jenkins job.
+BUILDPATH="${WORKSPACE}/go"
+mkdir -p $BUILDPATH
+cd $BUILDPATH
+export GOPATH="$( pwd )"
+WORKPATH="${BUILDPATH}/src/github.com/openshift/"
+mkdir -p $WORKPATH
+echo "GOPATH: ${GOPATH}"
+echo "BUILDPATH: ${BUILDPATH}"
+echo "WORKPATH ${WORKPATH}"
+
+# Ensure ssh-agent is running
+eval "$(ssh-agent -s)"
+
+
+cd ${WORKPATH}
+rm -rf ose
+git clone git@github.com:openshift/ose.git
+pushd ose
+git remote add upstream git@github.com:openshift/origin
+git fetch --all
+
+# Check if there is anything to update.
+# Tags are global to the git repo but can only be accessed from the branches they were tagged in
+# Switch to the enterprise-3.5 branch where our latest tags exist for now.
+git checkout enterprise-3.5
+LATEST_TAG=$(git describe --abbrev=0 --tags)
+LATEST_TITO_COMMIT_MERGE=$(git log -1 $LATEST_TAG --pretty=%s)
+LATEST_TITO_COMMIT_REBASE=$(git log -1 origin/fake-master --pretty=%s)
+if [[ $LATEST_TITO_COMMIT_MERGE == $LATEST_TITO_COMMIT_REBASE ]]; then
```

Comment:
I think we switched this in a version of this script in some other branch ( @smunilla ) but you can ask `git` to do this directly for you:

```sh
if git describe --abbrev=0 --tags --exact-match HEAD >/dev/null 2>&1; then
    # latest tag matches HEAD
```